hydra-chain-observer/src/Hydra/ChainObserver.hs

```diff
@@ -152,7 +153,7 @@ chainSyncClient tracer networkId startingPoint =
     ClientStNext
       { recvMsgRollForward = \blockInMode _tip -> ChainSyncClient $ do
           case blockInMode of
-            BlockInMode (Block _header txs) BabbageEraInCardanoMode -> do
+            BlockInMode BabbageEra (Block _header txs) BabbageEraInCardanoMode -> do
```

Comment:
Seeing this one really makes you wonder _"why another witness?"_.

The change here is fine, but this really makes we sad and want to stop using `cardano-api` right away.

You _could_ just not match on it like you did in the `hydra-node`
```suggestion
            BlockInMode _ (Block _header txs) BabbageEraInCardanoMode -> do
```